solr/server/solr/configsets/_default/conf/solrconfig.xml

```diff
@@ -416,8 +416,7 @@
                       to occupy. Note that when this option is specified, the size
                       and initialSize parameters are ignored.
       -->
```

Comment:
Could you fix this docs to state that the default cache class is `CaffeineCache`?